src/supertux/sector.cpp

```diff
@@ -720,21 +720,29 @@ Sector::get_gravity() const
 }
 
 Player*
-Sector::get_nearest_player (const Vector& pos) const
+Sector::get_nearest_player(const Vector& pos) const
 {
-  Player *nearest_player = nullptr;
+  auto players = get_objects_by_type_index(typeid(Player));
+  if (players.size() == 1)
+  {
+    Player* player = static_cast<Player*>(players[0]);
+    return ((player->is_dying() || player->is_dead()) ? nullptr : player);
```

Comment:
Remind me to remind you to use player->is_alive() once my player method PR is merged.